nilearn/tests/test_masking.py

```diff
@@ -580,80 +761,105 @@ def test__extrapolate_out_mask():
-                             [False, False, False, False, False],
-                             [False, False, False, False, False]]])
+    target_data = np.array(
+        [
+            [
+                [0.0, 0.0, 0.0, 0.0, 0.0],
+                [0.0, 0.0, 0.0, 0.0, 0.0],
+                [0.0, 0.0, 1.0, 0.0, 0.0],
+                [0.0, 0.0, 0.0, 0.0, 0.0],
+                [0.0, 0.0, 0.0, 0.0, 0.0],
+            ],
+            [
+                [0.0, 0.0, 0.0, 0.0, 0.0],
+                [0.0, 0.0, 1.5, 0.0, 0.0],
+                [0.0, 2.0, 1.0, 3.5, 0.0],
+                [0.0, 0.0, 3.0, 0.0, 0.0],
+                [0.0, 0.0, 0.0, 0.0, 0.0],
+            ],
+            [
+                [0.0, 0.0, 2.0, 0.0, 0.0],
+                [0.0, 2.5, 2.0, 4.0, 0.0],
+                [3.0, 3.0, 3.5, 6.0, 6.0],
+                [0.0, 4.0, 5.0, 5.5, 0.0],
+                [0.0, 0.0, 5.0, 0.0, 0.0],
+            ],
+            [
+                [0.0, 0.0, 0.0, 0.0, 0.0],
+                [0.0, 0.0, 3.0, 0.0, 0.0],
+                [0.0, 3.5, 4.0, 5.0, 0.0],
+                [0.0, 0.0, 4.5, 0.0, 0.0],
+                [0.0, 0.0, 0.0, 0.0, 0.0],
+            ],
+            [
+                [0.0, 0.0, 0.0, 0.0, 0.0],
+                [0.0, 0.0, 0.0, 0.0, 0.0],
+                [0.0, 0.0, 4.0, 0.0, 0.0],
+                [0.0, 0.0, 0.0, 0.0, 0.0],
+                [0.0, 0.0, 0.0, 0.0, 0.0],
+            ],
+        ]
+    )
+    target_mask = np.array(
+        [
+            [
+                [False, False, False, False, False],
+                [False, False, False, False, False],
+                [False, False, True, False, False],
+                [False, False, False, False, False],
+                [False, False, False, False, False],
+            ],
+            [
+                [False, False, False, False, False],
+                [False, False, True, False, False],
+                [False, True, True, True, False],
+                [False, False, True, False, False],
+                [False, False, False, False, False],
+            ],
+            [
+                [False, False, True, False, False],
+                [False, True, True, True, False],
+                [True, True, True, True, True],
+                [False, True, True, True, False],
+                [False, False, True, False, False],
+            ],
+            [
+                [False, False, False, False, False],
+                [False, False, True, False, False],
+                [False, True, True, True, False],
+                [False, False, True, False, False],
+                [False, False, False, False, False],
+            ],
+            [
+                [False, False, False, False, False],
+                [False, False, False, False, False],
+                [False, False, True, False, False],
+                [False, False, False, False, False],
+                [False, False, False, False, False],
+            ],
+        ]
+    )
```

Comment:
I think this is equivalent:

```suggestion
    target_mask = np.not_equal(target_data, 0)
```

Reply:
and only "just" a bit shorter. 🤩